docs/products/opensearch/concepts/access_control.rst

```diff
@@ -43,43 +61,65 @@ The same set would deny the service user to
 * read or search documents from ``events_2018`` (the second rule only grants ``write`` permission), and
 * write to or use the API for ``logs_20171230`` (the first rule only grants ``read`` permission).
 
-The permission also implies which index APIs the service user can access:
+Controlling access to top-level APIs
+------------------------------------
 
-* ``read``:  ``_search``, ``_mget``
-* ``write``: ``_bulk``, ``_mapping``, ``_update_by_query``, ``_delete_by_query``
-* ``admin``: no restrictions 
+In addition to indices, ACLs can be used to control access to "top-level" APIs by creating a API specific rule.
 
-  
+.. note::
+   Access to the ``_cluster``, ``_cat``, ``_tasks``, ``_scripts``, ``_snapshot``, and ``_nodes`` APIs is controlled
+   by the service itself, not by these ACLs.
 
-Controlling access to top-level APIs
-------------------------------------
+.. note::
+    Access to the top-level ``_mget``, ``_msearch``, and ``_bulk`` APIs can also be controlled by enabling ``ExtendedACL``.
```

Comment:
Do we have more on the ExtendedACLs? a link would clarify what can be done

Reply:
We don't have much more information on ExtendedACL. And ACL stuff for OpenSearch is evolving, so there are improvements to come.